packages/wrangler/src/is-interactive.ts

```diff
@@ -1,13 +1,13 @@
 import { isInteractive as __isInteractive } from "@cloudflare/cli/interactive";
-import { CI } from "./is-ci";
+import { CI, isPagesCI, isWorkersCI } from "./is-ci";
 
 /**
  * Test whether the process is "interactive".
  * Reasons it may not be interactive: it could be running in CI,
  * or you're piping values from / to another process, etc
  */
 export default function isInteractive(): boolean {
-	if (process.env.CF_PAGES === "1") {
+	if (isPagesCI() || isWorkersCI()) {
```

Comment:
q: without knowing the context I would use `CI.isCI()`. Maybe add a comment on why it is not used.